docs/catalog/sinks/PagerDuty.rst

```diff
@@ -97,3 +97,19 @@ Example Output:
     .. image:: /images/change-events-updated-deployment-pagerduty.png
       :width: 1000
       :align: center
+
+Support
+----------------------
+
+Ask for help, or just say hi!
+
+1. `Slack <https://bit.ly/robusta-slack>`_ - **link only works on desktop**
+2. `GitHub Issues <https://github.com/robusta-dev/robusta/issues>`_
+
+
+**Commercial Support**
```

Comment:
Can we remove this part?

Reply:
Will `3.  Email: support@robusta.dev` do?

Reply:
Sure.

Reply:
@aantn done!

Reply:
If the changes look good, i will merge it.

Reply:
I pushed the button for you :)

Reply:
Thank you!🙌🙌😌